workspaces/sonarqube/plugins/sonarqube-backend/src/service/sonarqubeInfoProvider.ts

```diff
@@ -108,6 +108,10 @@ export interface SonarqubeInstanceConfig {
    * Access token to access the sonarqube instance as generated in user profile.
    */
   apiKey: string;
+  /**
+   * The type of API Key used in apiKey. Can be Bearer or Basic. Defaults to Basic if not set or invalid.
+   */
+  tokenKind?: string;
```

Comment:
```suggestion
  tokenKind: 'Bearer' | 'Basic';
```

If we do that then won't that also allow us to removed the `convertTokenKind` function?